src/WCSimConstructCylinder.cc

```diff
@@ -276,8 +275,7 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCylinder()
 					  false,
 					  checkOverlaps);
 
-	//G4LogicalSkinSurface *TyvekCaveBarrelSurface =
-	new G4LogicalSkinSurface("TyvekCaveBarrelSurface", logicCaveTyvek, OpWaterTySurface);
+    G4LogicalSkinSurface *TyvekCaveBarrelSurface = new G4LogicalSkinSurface("TyvekCaveBarrelSurface", logicCaveTyvek, OpWaterTySurfaceOutWallBarrel);
```

Comment:
General comment for this file: don't declare variables that you don't need to use. We currently have a build that is clear of compilation warnings - I'd like to keep it that way
```suggestion
    new G4LogicalSkinSurface("TyvekCaveBarrelSurface", logicCaveTyvek, OpWaterTySurfaceOutWallBarrel);
```

Reply:
you are right, thanks, I fixed them all